target/linux/ramips/dts/mt7621_netgear_r6800.dts

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "mt7621_netgear_sercomm_bzv.dtsi"
+
+/ {
+	compatible = "netgear,r6800", "mediatek,mt7621-soc";
+	model = "Netgear R6800";
+
+	aliases {
+		led-boot = &led_power_white;
+		led-failsafe = &led_power_orange;
+		led-running = &led_power_white;
+		led-upgrade = &led_power_orange;
+		label-mac-device = &ethernet;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power_orange: power_orange {
+			gpios = <&gpio_expander 0 GPIO_ACTIVE_LOW>;
+			label = "r6800:orange:power";
+		};
+
+		led_power_white: power_white {
+			gpios = <&gpio_expander 1 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:power";
+		};
+
+		wan_orange {
+			gpios = <&gpio_expander 2 GPIO_ACTIVE_LOW>;
+			label = "r6800:orange:wan";
+		};
+
+		wan_white {
+			gpios = <&gpio_expander 3 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:wan";
+		};
+
+		wifi2g {
+			gpios = <&gpio_expander 4 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:wlan2g";
+			linux,default-trigger = "phy0tpt";
+		};
+
+		wifi5g {
+			gpios = <&gpio_expander 5 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:wlan5g";
+			linux,default-trigger = "phy1tpt";
+		};
+
+		usb2 {
+			gpios = <&gpio_expander 6 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:usb2";
+			linux,default-trigger = "usbport";
+			trigger-sources = <&hub_port2>;
+		};
+
+		usb3 {
+			gpios = <&gpio_expander 7 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:usb3";
+			linux,default-trigger = "usbport";
+			trigger-sources = <&hub_port1>;
+		};
+
+		lan1_orange {
+			gpios = <&gpio_expander 8 GPIO_ACTIVE_LOW>;
+			label = "r6800:orange:lan1";
+		};
+
+		lan1_white {
+			gpios = <&gpio_expander 9 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:lan1";
+		};
+
+		lan2_orange {
+			gpios = <&gpio_expander 10 GPIO_ACTIVE_LOW>;
+			label = "r6800:orange:lan2";
+		};
+
+		lan2_white {
+			gpios = <&gpio_expander 11 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:lan2";
+		};
+
+		lan3_orange {
+			gpios = <&gpio_expander 12 GPIO_ACTIVE_LOW>;
+			label = "r6800:orange:lan3";
+		};
+
+		lan3_white {
+			gpios = <&gpio_expander 13 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:lan3";
+		};
+
+		lan4_orange {
+			gpios = <&gpio_expander 14 GPIO_ACTIVE_LOW>;
+			label = "r6800:orange:lan4";
+		};
+
+		lan4_white {
+			gpios = <&gpio_expander 15 GPIO_ACTIVE_LOW>;
+			label = "r6800:white:lan4";
+		};
+
+		wps {
+			gpios = <&gpio0 17 GPIO_ACTIVE_HIGH>;
+			label = "r6800:white:wps";
+		};
+
+		wifi {
```

Comment:
What's the purpose of this LED in addition to the two WiFi leds above? Also consider that for proper choice of names ...

Reply:
Router have 3 wifi LEDs: Wifi 5Ghz, Wifi 2,4Ghz and Wifi on/off LED.

Reply:
Hmm, okay, then maybe just keep "wifi" or "wlan" for this one.
Maybe add the comment to commit message:
"Device has 3 wifi LEDs: Wifi 5Ghz, Wifi 2.4Ghz and Wifi on/off"